### PR TITLE
Clarify file numbers produced in ISIS SANS Manual Tests

### LINKED ISSUE #37373: Update SANS Manual Testing Output Instructions
**Describe the issue**
- The SANS GUI only outputs the Can calculated workspace for the first workspace if they all use the same ones. The manual testing docs don't mention this. 

- [x] Update manual testing to clarify this.

### PR DESCRIPTION
**Pull request by cailafinn**
RE #37373

### Description of work
Amend the manual tests for SANS to point out that the `can` file only gets produced once during the reduction. This is due to the can calculation being the same for both rows of the table. 

Fixes #37373 

### To test:
1. Build the dev-docs-html target
2. Check grammar & formatting. 
1. Follow the new instructions in the changed Processing -> 1D reduction section. 
2. Check the number of produced files is correct. 

*This does not require release notes* because **it only affects dev docs.**

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

**Review by adriazalvarez — DISMISSED** [review]
Followed instructions. The output of the testing exercise is more understandable.
Thanks for improving the instructions.

File: dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst [hunk 1]
```diff
@@ -124,8 +124,10 @@ Processing
 #. In the workspaces list, there should be a series of new workspaces; four
    group workspaces and four 1D workspaces.
 #. Check your default save directory. For each reduction two banks (HAB/main) should
-   be saved. In total there should be 12 workspaces (6 CanSAS ``.xml`` and 6 NXcanSAS ``.h5``) saved. For each row and
-   file type there should be a reduced file (with no suffix), a sample, and a can workspace.
+   be saved. In total there should be 20 workspaces saved. For each row, file type, and bank there should be a
```
Comment (SilkeSchomann) [inline dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst:127]: be saved. In total, there should be 20 workspaces saved. For each row, file type, and bank, there should be a

Comment (SilkeSchomann) [reply dev-docs/source/Testing/SANSGUI/ISISSANSGUITests.rst:127]: I’m aware that I’m nit-picking, and I apologize for it.

**Review by SilkeSchomann — CHANGES_REQUESTED** [review]

**Review by SilkeSchomann — APPROVED** [review]
Thanks for making these changes!

**Pull request merged**